guides/source/working_with_javascript_in_rails.md

```diff
@@ -505,20 +507,22 @@ attribute to the tag:
 ### Page Change Events
 
 When writing CoffeeScript, you'll often want to do some sort of processing upon
-page load. With jQuery, you'd write something like this:
+page load. With vanilla JavaScript, you'd write something like this:
```

Comment:
```suggestion
page load. Using the DOM, you'd write something like this:
```
Also bonus points if we replace our first reference to the DOM with: `[Document Object Model (DOM)](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction)`